docs/getting-started.md

```diff
@@ -38,6 +38,8 @@ Lastly, install the React Native for Windows packages.
 npx react-native-windows-init --overwrite
 ```
 
+> The --overwrite flag is a temporary measure that ensures the correct files are copied to metro.config.js for the metro bundler to work with Windows. If you are starting a new app, this should have no impact. If you are adding Windows to your existing app and you have modified the metro.config.js file, please back up your changes, run the command and copy over to take effect.
+
 #### Confirming Metro JS reinstallation

```

Comment:
What is this section header pointing to? I dont see any content under it?

Reply:
it's not a section header, ">" are inline notes.

Reply:
Sorry i was referring to "confirming Metro JS reinstallation"

Reply:
oh whoops! this was an artifact left over from an edit on this pr. Thanks for catching, I'll remove.